src/panels/calendar/ha-panel-calendar.ts

```diff
@@ -0,0 +1,304 @@
+import {
+  customElement,
+  LitElement,
+  property,
+  CSSResultArray,
+  css,
+  TemplateResult,
+  html,
+} from "lit-element";
+import { styleMap } from "lit-html/directives/style-map";
+
+import "@polymer/app-layout/app-header-layout/app-header-layout";
+import "@polymer/app-layout/app-header/app-header";
+import "@polymer/app-layout/app-toolbar/app-toolbar";
+import "@polymer/paper-checkbox/paper-checkbox";
+
+import "../../components/ha-menu-button";
+import "../../components/ha-card";
+import "../../components/ha-checkbox";
+import "./ha-full-calendar";
+
+import type {
+  HomeAssistant,
+  SelectedCalendar,
+  CalendarEvent,
+  Calendar,
+  CalendarViewChanged,
+} from "../../types";
+import { haStyle } from "../../resources/styles";
+import { HASSDomEvent } from "../../common/dom/fire_event";
+
+const palette = [
```

Comment:
Not sure if this is even a good idea to use this color list. I think the best option would be allowing to set your own colors in the config. But maybe this is good for now? Also, if it doesn't have a color we use this? Also, this is copied so I need to put this some where and import it